src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -81,14 +83,16 @@ const en = ({ isDivorce, userCase, partner }: CommonContent) => ({
     )} after your application has been checked.
     This will have the time, date and court your conditional order will be pronounced.`,
   },
-  subHeading1:
-    userCase.coClarificationUploadDocuments || userCase.coClarificationResponses
-      ? 'Latest information'
-      : 'What you need to do',
+  awaitingFinalOrder: {
+    line1: `You can now apply for a ‘final order’. A final order is the document that will legally end your ${
+      isDivorce ? 'marriage' : 'civil partnership'
+    }. It’s the final step in the ${isDivorce ? 'divorce process' : 'process to end your civil partnership'}.`,
+  },
 });
 
 // @TODO translations
```

Comment:
I might be mistaken but looks like all the Welsh is there now, so we could delete this?

Reply:
Agree - will get rid